### DISCUSSION TITLE
New degree name (Data Science and Advanced Analytics)

### DISCUSSION THREAD

**Discussion by ph1001** [joaomlourenco/novathesis#305] [Q&A]
Solved:

Hello everyone,

First, big thanks to the authors of this template!

Secondly, I am wondering: Is there any way to change the appearance of the file nova-ims-mmaa-cover.pdf? I am asking because the master's degree Advanced Analytics has changed its name to Data Science and Advanced Analytics. It would be very nice if you could provide the means needed for creating this file

**Comment by ph1001** [comment]
I just found the answer myself. It's incredibly simple: It can be edited with Word.

I tried to delete this entry but I think it's not possible

**Reply by joaomlourenco** [reply]
@ph1001  Thank you for letting me know the Master's has a new name.  Now, just to understand better your own solution… what did you edit with word?  Thanks.

**Reply by ph1001** [reply]
I downloaded nova-ims-mmaa-cover.pdf (from novathesis/NOVAthesisFiles/Schools/nova/ims/Images), opened it in Word, changed the course's name, saved it as pdf, and replaced the file (in novathesis/NOVAthesisFiles/Schools/nova/ims/Images) with my edited version
